public/historyexamine.php

```diff
@@ -152,7 +161,7 @@ function convertDate() {
             echo "</td>";
 
             echo "<td nowrap>";
-            echo "$achPoints $achPointsNote";
+            echo "$achPointsNote";
```

Comment:
The summary (line 182) still shows combined totals.

history list:
![image](https://user-images.githubusercontent.com/32680403/174443931-0f7c84da-a846-4b6c-8f09-9c6cfe0e3566.png)
drilldown summary:
![image](https://user-images.githubusercontent.com/32680403/174443944-87111055-166e-420f-92c4-2da2331fb5e2.png)